src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/containers/ContainerManager.java

```diff
@@ -231,8 +230,27 @@ public int runSteps(
     return Job.STATUS_OK;
   }
 
+  public ContainerExecResponse runParameterisation(
+      TaskCopy c,
+      File codeDir,
+      RepoInfo repoInfo)
+      throws ApiUnavailableException {
+    if (!c.getDetail().getParameterisation().isPresent()) {
+      return null;
```

Comment:
Is null the right thing here? Would it be better to return a ContainerExecResponse that shows trivial success?

Reply:
If we return a success it'll get stored as the problemStatement so I'm not sure if it's a good idea. And this is only meant to be called if there is a parameterisation.

I've added a checkNotNull instead as this should only be called if there is a parameterisation. And also on the execution.

Reply:
ok thanks